Latest/Model/AppLibrary.swift

```diff
@@ -47,20 +47,43 @@ class AppLibrary {
 	@objc func gatherAppBundles() {
 		self.appSearchQuery.disableUpdates()
 		
-		// Run metadata query to gather all apps
-		self.bundles = self.appSearchQuery.results.compactMap { item -> App.Bundle? in
-			guard let metadata = item as? NSMetadataItem,
-				let path = metadata.value(forAttribute: NSMetadataItemPathKey) as? String else { return nil }
-			
-			let url = URL(fileURLWithPath: path)
-			
-			// Only allow apps in the application folder and outside excluded subfolders
-			if !Self.applicationPaths.contains(where: { url.path.hasPrefix($0) }) || Self.excludedSubfolders.contains(where: { url.path.contains($0) }) {
-				return nil
+		let appSearchQueryResults = self.appSearchQuery.results
+		if appSearchQueryResults.isEmpty {
+			// Look for applications in the /Applications folder manually
+			let fileManager = FileManager.default
+			var bundles = [App.Bundle]()
+			do {
+				for applicationPath in Self.applicationPaths {
+					let appPaths = try fileManager.contentsOfDirectory(atPath: applicationPath)
+					for appPath in appPaths {
+						let url = URL(fileURLWithPath: applicationPath + "/" + appPath)
+						if !Self.applicationPaths.contains(where: { url.path.hasPrefix($0) }) || Self.excludedSubfolders.contains(where: { url.path.contains($0) }) {
+							continue
+						}
+						if let bundle = self.bundle(forAppAt: url) {
+							bundles.append(bundle)
+						}
+					}
+				}
+			} catch {
```

Comment:
Since we currently don't have a way to handle these errors, I think it is fine to ignore them. You could remove the do/catch blocks by using `try?` instead of `try`. See my other comments, you may not need this at all :)

Reply:
Done